autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -117,6 +117,20 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
             Use `auto_stack` to maximize predictive accuracy; use `hyperparameter_tune` if you prefer to deploy just a single model rather than an ensemble.
         feature_prune : bool, default = False
             Whether or not to perform feature selection.
+        refit_full : bool, default = False
+            Whether to retrain all models on all of the data after the normal training procedure.
+            The time taken by this process is not enforced by `time_limits`.
+            If `True`, optimizes a model's inference time by collapsing bagged ensembles into a single model fit on all of the training data.
+            This process will typically result in a slight accuracy reduction and a large inference speedup.
+            The inference speedup will generally be between 10-200x faster than the original bagged ensemble model.
+            The added fit runtime of this function is generally 10% or less of the original fit runtime.
+            This function can also be called on non-bagged models for a slight accuracy increase and no change to inference time.
+                In this instance, the added fit runtime will be approximately equal to the original fit runtime.
+            This process does not alter the original models, but instead fits additional models.
+            Models produced by this process will not have validation scores, as they use all of the data for training.
+                Therefore, it is up to the user to determine if the models are of sufficient quality by including test data in `predictor.leaderboard(dataset=test_data)`.
+            This is equivalent to calling `predictor.refit_full()` after training.
```

Comment:
The documentation for `predictor.refit_full()` is missing the case of non-bagged ensembles.  In general both docs should probably be more clearly partitioned into two sections: bagging was used in task.fit() vs bagging was not used.

Reply:
Added